From 458d1a661ab9c6c4f0d1390c7f5f8580f6cd1390 Mon Sep 17 00:00:00 2001
From: Dan Kalowsky <daniel.kalowsky@intel.com>
Date: Tue, 9 Sep 2014 14:25:47 -0700
Subject: [PATCH] intel_idle: Removing C1E on BYT

Removing the use of C1E on BYT-T-CR.  It's questionable if the C1E
actually provides any power advantage and more accurately is believed to
impact performance.

Change-Id: I7a1a82157710cab7069f953ce4a2dd7702df3103
Signed-off-by: Dan Kalowsky <daniel.kalowsky@intel.com>
(cherry picked from commit d2ca45bfdeb90155a8618a48f750c6b86c4b0e0a)

Signed-off-by: Nagaraj S <sivasankaranx.nagaraj@intel.com>
---
 drivers/idle/intel_idle.c |    7 -------
 1 file changed, 7 deletions(-)

diff --git a/drivers/idle/intel_idle.c b/drivers/idle/intel_idle.c
index 25b0218..3896e8e 100644
--- a/drivers/idle/intel_idle.c
+++ b/drivers/idle/intel_idle.c
@@ -242,13 +242,6 @@ static struct cpuidle_state byt_cstates[] = {
 		.target_residency = 1,
 		.enter = &intel_idle },
 	{
-		.name = "C1E-BYT",
-		.desc = "MWAIT 0x01",
-		.flags = MWAIT2flg(0x01) | CPUIDLE_FLAG_TIME_VALID,
-		.exit_latency = 15,
-		.target_residency = 30,
-		.enter = &intel_idle },
-	{
 		.name = "C6N-BYT",
 		.desc = "MWAIT 0x58",
 		.flags = MWAIT2flg(0x58) | CPUIDLE_FLAG_TIME_VALID | CPUIDLE_FLAG_TLB_FLUSHED,
-- 
1.7.9.5

